wgsl/index.bs

```diff
@@ -1439,6 +1439,81 @@ Details of conversion to and from floating point are explained in [[#floating-po
 
 </table>
 
+## Reinterpretation of Representation Expressions ## {#bitcast-expr}
+
+A `bitCast` expression is used to reinterpet the bit representation of a
```

Comment:
I believe this should be `bitcast` after the last meeting?

Reply:
I've fixed all instances of bitCast to bitcast.  Thanks!